src/main/java/se/sundsvall/supportmanagement/api/model/errand/Errand.java

```diff
@@ -89,31 +92,31 @@ public class Errand {
 	private String escalationEmail;
 
 	@Schema(description = "Contact reason for the errand", example = "The printer is not working")
-	@ValidContactReason(groups = {OnCreate.class, OnUpdate.class}, nullable = true)
+	@ValidContactReason(groups = { OnCreate.class, OnUpdate.class }, nullable = true)
 	private String contactReason;
 
 	@Schema(description = "Suspension information")
 	@Valid
 	private Suspension suspension;
 
 	@Schema(description = "Flag to indicate if the errand is business related", example = "true")
-	//TEMPORARY FIX: This should be set to true in version 6.0
-	//@NotNull(groups = {OnCreate.class, OnUpdate.class})
+	// TEMPORARY FIX: This should be set to true in version 6.0
+	// @NotNull(groups = {OnCreate.class, OnUpdate.class})
```

Comment:
Not related to this PR really but as this goes to 6.0 this should be activated.

Reply:
Ok, "activated" now.